frontend/src/components/SubstitutionDate.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+import ErrorMessage from "./ErrorMessage";
+
+const SubstitutionDateWrapper = styled.div`
+  position: fixed; 
+  bottom: 0;
+  right: 0;
+`;
+
+const SubstitutionDate = () => {
+  const [date, setDate] = useState("");
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    axios
+      .get(`/api/date`)
+      .then(({ data }) => setDate(data.date))
+      .catch((err) => setError(String(err)));
+  }, []);
+
+  return (
+    <>
+      <ErrorMessage error={error} />
```

Comment:
I don't think this component is appropriate here.
![image](https://user-images.githubusercontent.com/19212948/74877657-17d7a700-5366-11ea-9976-e57c2a220e9f.png)
This makes this big error message and still, there is this text with the empty date.

Reply:
Also, this 'date' is for the user to be sure that he is looking at up-to-date substitutions. Here are possible scenarios when an error might occur:

1. teacher and date endpoint returned error - This is pretty obvious, the app is down, therefore an error message will be displayed instead of teacher list.

2. teacher endpoint returned error but date endpoint works fine - The user will see an error message instead of the teacher list, so he knows that the app is broken, even that date is showed in the corner.

3. teacher and date endpoint returned valid data, but the date is out-of-date - it's not actually an error, but based on that date the user will notice that there is something wrong.

4. that last might be a little odd: the backend returned the list of substitute teachers but returned an error on date endpoint - I'm not sure what should we do in this situation, because this probably means that format of the html file has changed. So the list of substitutions is probably up-to-date. Maybe we should put a text "Couldn't get the date" or something like that